security/appsec/intro/what-else-do-you-need-to-know.md

```diff
@@ -55,7 +55,7 @@ How memory in a computer works, and where data is located at any given time
 ---
 ## Revision
 
-If you'd like to learn more about application security, you should make sure you have a strong grounding on the following:  
+If you'd like to learn more about application security, you should make sure you have a strong grounding in the following:  
```

Comment:
should we use "foundation" instead of "grounding"? Seems to me like a more common term

Reply:
```suggestion
If you'd like to learn more about application security, you should make sure you have a strong foundation in the following:  
```